app/javascript/app/phone-internationalization.js

```diff
@@ -104,7 +106,7 @@ const updateInternationalCodeSelection = () => {
   const internationalCode = internationalCodeFromPhone(phone);
   const option = document.querySelector(`[data-country-code='${internationalCode}']`);
   if (option) {
-    const dropdown = document.querySelector('[data-international-phone-form] .international-code');
+    const dropdown = $('#user_phone_form_international_code');
```

Comment:
Why was Code Climate complaining about this?

Reply:
This was from when I looking to simplify this JS and use the ID selector instead

Reply:
I see. And is there a reason for using jQuery here instead of `document.querySelector`? I'm just thinking ahead for when we eventually remove the jQuery dependency.

Reply:
When I change it from using jQuery to using `document.querySelector` the international select field (which is hidden) no longer changes along with country code. 

<img width="396" alt="screen shot 2018-05-15 at 12 34 17 pm" src="https://user-images.githubusercontent.com/1449852/40125702-890dd228-58f9-11e8-8d05-35c385887375.png">

Reply:
I'm disabling the CSS that is hiding it. By default it is hidden.

Reply:
Why do we care about updating the hidden field?

Reply:
Also, I just tried to locally with `document.querySelector('#user_phone_form_international_code');` and the hidden field is updating for me.

Reply:
I could be wrong but I thought the international code field was still being sent when submitting the form

Reply:
It works in both Chrome and Safari for me.